packages/node_modules/@ciscospark/react-component-typing-avatar/src/index.js

```diff
@@ -20,17 +20,10 @@ const defaultProps = {
 };
 
 function TypingAvatar({avatarId, isTyping, name}) {
-  let typingIndicator;
-  if (isTyping) {
-    typingIndicator = <TypingIndicator />;
-  }
   return (
     <div className={classNames('ciscospark-typing-avatar', styles.typingAvatar)} data-tooltip={name}>
       <div className={classNames('ciscospark-avatar', styles.avatar)}>
-        <PresenceAvatar avatarId={avatarId} iconSize="20px" name={name} />
-      </div>
-      <div className={classNames('ciscospark-typing', styles.typing)}>
```

Comment:
Can we remove the `.typing` class from the css?

Reply:
oops, yes x 3 :)